### DISCUSSION TITLE
I've added GitHub Student benefits to WRONG account

### DISCUSSION THREAD

**Discussion by ghost** [community/community#46890] [GitHub Education]
Hello dear supporters! 👋

Today, I made a mistake, in which I had **linked my school email to this GitHub account** to get the **Student Benefits**, but after I linked, I realised that this is **my secondary account**. Can I transfer the benefits to my principal account, and also remove them from this account? 

Thank you! Have a nice day/night! 💗

**Comment by chris-c-thomas** [comment]
I'm unsure in regards to the way you suggest. I do know that GitHub really does recommend having only one account so you could merge your other account into this one? Not sure your needs, however.

Personal, professional, school, etc. Your GitHub account is meant to be tied to you as a person and can support all the emails and be configured with different emails for different repos/orgs. If you follow [these docs](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-personal-account/merging-multiple-personal-accounts) your commit history from your other account will follow. Or the other way around.

I'm just not sure on the benefits and support may need to chime in on that.

**Reply by D3sTeptu** [reply]
Yes, but I mean, I linked this account instead of my main account: D3sTeptu, and I want to transfer all the benefits to that account

**Reply by chris-c-thomas** [reply]
Right, I understand. But you should consider merging the two accounts (as in one username) with all your emails tied to that account. Commits and history follow email addresses. 

Here in the [docs](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/apply-to-github-global-campus-as-a-student) says that you cannot transfer academic discounts including the student pack. It suggests merging instead.

![image](https://user-images.githubusercontent.com/22353987/217916207-4e168b7f-7623-42c0-8e81-c26b5b4236ff.png)

**Comment by ghost** [comment]
Oh, ok! I will try! Thank you also!! ❤

**Reply by chris-c-thomas** [reply]
No problem! You can also change your username if desired, afterwards.

**Reply by D3sTeptu** [reply]
All right! 👍

**Reply by nadekoi** [reply]
Did you tried it and did it worked ? I got the same problem and I would want to know if it worked for you. Thanks !

**Comment by Sidd-17052004** [comment]
> Hello dear supporters! 👋
> 
> Today, I made a mistake, in which I had **linked my school email to this GitHub account** to get the **Student Benefits**, but after I linked, I realised that this is **my secondary account**. Can I transfer the benefits to my principal account, and also remove them from this account?
> 
> Thank you! Have a nice day/night! 💗

I have also made same problem. What if I unlink my student mail and add to my primary account. Will my student benefits be there . Please tell fast. It will be very helpful

**Reply by A-Shalchian** [reply]
It wouldn't work, as I've already tried that. I have two accounts one with the student benefit and one is my main. if you remove the email and add it to your primary account, it would work, BUT when you go to the GitHub education page it would give you an error of being on the other account. Now maybe you can open a ticket/ what GitHub says merge the accounts and then rename it if you want.

**Reply by FarhanKhai** [reply]
You mean you didn't get the student benefits in the GitHub profile you want?

**Comment by ghost** [comment]
> Hello dear supporters! 👋
> 
> Today, I made a mistake, in which I had **linked my school email to this GitHub account** to get the **Student Benefits**, but after I linked, I realised that this is **my secondary account**. Can I transfer the benefits to my principal account, and also remove them from this account?
> 
> Thank you! Have a nice day/night! 💗

Has this worked out for you? I'm trying to get the benefits for my main account, is there any way what so ever? I'm stuck having the benefits of the GitHub student on an account that I don't really want to use (because the email name is embarassing)
